### PR TITLE
Add alert ServiceLostHA in prow-monitoring

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

File: cluster/ci/monitoring/Makefile [hunk 1]
```diff
@@ -31,6 +29,17 @@ prow-monitoring-deploy:
 	oc apply -f ./grafana_deploy.yaml
 	oc apply -f ./grafana_service_monitor.yaml
 
+slack_api_url := $(shell oc get secret -n ci ci-slack-api-url -o json | jq -r .data.url | base64 -d)
+
+.PHONY : ci-secret-deploy
+ci-secret-deploy:
+	@echo "deploy ci secret ..."
+	#TODO: reference a value should be doable with jsonnet
+	sed -e "s|{{ api_url }}|$(slack_api_url)|g" ./alertmanager.yaml > /tmp/alertmanager_with_secret.yaml
```
Comment (stevekuznetsov) [inline cluster/ci/monitoring/Makefile]: Why can't the alertmanager mount the secret as a volume?

Comment (hongkailiu) [reply cluster/ci/monitoring/Makefile]: The secret created by `bitwarden.sh` is only the api-url for slack and it is only _a field_ of the config file `./alertmanager.yaml` which is mounted in the pod via secret `alertmanager-prow`.

**Comment by hongkailiu** [comment]
/hold cancel
Upstream not supported yet.

**Pull request merged**